src/args.ts

```diff
@@ -7,14 +7,14 @@ export function parseArgs(args: string[]) {
     alias: {
       "help": "h",
       "version": "V",
-      "project": "p",
```

Comment:
why is this alias removed?

Reply:
Oh! it's a mistake. I added a bunch of alias and then I changed my mind. I must had removed one too many